client/my-sites/plans-grid/hooks/npm-ready/data-store/use-plan-features-for-grid-plans.ts

```diff
@@ -68,6 +74,31 @@ const usePlanFeaturesForGridPlans: UsePlanFeaturesForGridPlans = ( {
 					allFeaturesList,
 					planConstantObj.getBlogOnboardingSignupJetpackFeatures?.() ?? []
 				);
+			} else if ( 'plans-woocommerce' === intent ) {
```

Comment:
It's good to have these enclosed at the top level like this. This may turn into store selectors down the line (`getFeaturesForWooCommercePlans`, `getFeaturesForNewsletterPlans`, etc.)